dffml/df/base.py

```diff
@@ -184,6 +192,45 @@ def op(*args, imp_enter=None, ctx_enter=None, config_cls=None, **kwargs):
     iterated over and the values in that ``dict`` are entered. The value yielded
     upon entry is assigned to a parameter in the ``OperationImplementation``
     instance named after the respective key.
+
+    Examples
+    --------
+
+    >>> from dffml import Definition, Input, op
+    >>> from typing import NamedTuple, List, Dict
+    >>>
+    >>> class Person(NamedTuple):
+    ...     name: str
+    ...     age: int
+    ...
+    >>> @op
+    ... def isTeen(p: List[Person]):
+    ...     return [True if person.age > 18 else False for person in p]
+    ...
+    >>>
+    >>> Input(
+    ...     value=[
+    ...         {"name": "Bob", "age": 20},
+    ...         {"name": "Mark", "age": 21},
+    ...         {"name": "Alice", "age": 90},
+    ...     ],
+    ...     definition=isTeen.op.inputs["p"],
+    ... )
+    Input(value=[Person(name='Bob', age=20), Person(name='Mark', age=21), Person(name='Alice', age=90)], definition=isTeen.p)
+    >>> @op
+    ... def canVote(p: Dict[str, Person]):
+    ...     return [True if person.age > 18 else False for name,person in p.items()]
```

Comment:
Black formatting and consistency with above

```suggestion
    ...     return {
    ...         person.name: person
    ...         for person in filter(lambda person: person.age >= 18, p.values())
    ...     }
```